### DISCUSSION TITLE
"Delivered as raw message" format?

### DISCUSSION THREAD

**Discussion by sinnbeck** [postalserver/postal#3110] [Help with using Postal]
Does anyone know what format the "Delivered as raw message" http route setting is sending. I assumed it the "message" part would be an eml file, but I cannot open it.

It looks like this
![image](https://github.com/user-attachments/assets/6e5e5aae-eec5-40a0-aaee-063f8866f023)

I have considered it being due to some sort of encoding, but I cannot figure out what that would be then

**Comment by sinnbeck** [comment]
I changed it from json to form data and now it apparently works as expected. Not sure why

**Reply by willpower232** [reply]
Interesting, maybe some part of the message confused whatever Postal is using to make it JSON.

**Reply by sinnbeck** [reply]
Not quite sure. Could also be me messing up. I used webhook.site to get the data and copy pasted the base64 and decoded that. Perhaps their site changed it someone. Not sure :) But ayways it works now
